### PR TITLE
cleanup, add tests

### PR DESCRIPTION
**Pull request by jquense**
I don’t know why this commit was lost in https://github.com/react-bootstrap/react-bootstrap/pull/2252

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports = config => {
         }),
       ],
       devtool: 'cheap-module-inline-source-map',
+      stats: { modules: false },
```
Comment (taion) [inline karma.conf.js]: what's this?

File: src/ToggleButtonGroup.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import * as PropTypes from 'prop-types';
```
Comment (taion) [inline src/ToggleButtonGroup.js]: import PropTypes

File: karma.conf.js (hunk 1 above)
Comment (jquense) [inline karma.conf.js]: /shrug

File: src/ToggleButtonGroup.js [hunk 3]
```diff
@@ -78,9 +78,11 @@ class ToggleButtonGroup extends React.Component {
       'is set to "radio"'
     );
 
+    delete props.onChange;
```
Comment (taion) [inline src/ToggleButtonGroup.js]: delete `value` as well?

**Comment by taion** [comment]
I have a few more changes to fix up the examples a bit, go ahead and merge when you can

**Pull request merged**